Jenkinsfile

```diff
@@ -1,5 +1,6 @@
 #!/bin/groovy
 
+env.KBN_ES_SNAPSHOT_USE_UNVERIFIED = 'true'
```

Comment:
I know you're aware, just a reminder not to forget to remove this before merging!

Reply:
Will this PR pass without running against the unverified snapshot?

Reply:
I skimmed through the changes and appears this PR does expect authentication_type. Thoughts on skipping the tests for now so we can promote the snapshot and we can move forward with this change the re-enabling the tests?

Reply:
@tylersmalley this PR requires the unverified snapshot to pass, so if you need to skip these in order to promote the snapshot, then that's fine with me. I'll update this PR to unskip as needed

Reply:
Skipping is in progress here: https://github.com/elastic/kibana/pull/76041/files